dbbackup/management/commands/dbbackup.py

```diff
@@ -29,6 +29,8 @@ class Command(BaseDbBackupCommand):
         make_option("-s", "--servername", help="Specify server name to include in backup filename"),
         make_option("-z", "--compress", help="Compress the backup files", action="store_true", default=False),
         make_option("-e", "--encrypt", help="Encrypt the backup files", action="store_true", default=False),
+        make_option("-m", "--mongo", help="Backup a mongo db instead of sql. Uses MONGO_SETTINGS", action="store_true",
+                    default=False),
```

Comment:
Since DATABASES is a dict that can contain many connections, I don't see why this isn't going the conventional db way.

The custom `temp_dir` can be added to that dict as well.?

```python
DATABASES = {
    'default': {...},
    'my_mongo': {
        'USER': 'dumper_user',
        'PASSWORD': '******',
        'ENGINE': 'mongo',
        'NAME': 'db_to_dump',
        'HOST': 'localhost',
        'PORT': '27017',
        'BACKUP_OPTIONS': {'temp_dir': '...'}
    }
}
```